engine/src/card.cpp

```diff
@@ -1088,30 +1088,43 @@ void MCCard::timer(MCNameRef mptr, MCParameter *params)
 		MCObject::timer(mptr, params);
 }
 
+bool MCCard::isdeletable()
+{
+    bool t_deletable = true;
+    
+    if (t_deletable)
+        t_deletable = MCObject::isdeletable();
+    
+    if (t_deletable)
+        t_deletable = !getflag(F_C_CANT_DELETE);
```

Comment:
Hmm... the revert command has no documented interaction with cantDelete however it would have been failing silently previously on stacks with cantDelete true and the other previously checked conditions in `del()`:

```
Boolean oldlock = MClockmessages;
    MClockmessages = True;
    MCerrorlock++;
    if (p_stack->del())
        p_stack -> scheduledelete();
    MCerrorlock--;
    MClockmessages = old lock;
```

Actually I'm wondering if revert silently failing is one of the reasons for _stack name is already loaded hell_ syndrome because it continues after the silent failure of `del()` to load the stack from filename.

```
p_stack = MCdispatcher->findstackname(*t_name);
    if (p_stack != NULL)
        p_stack->openrect(oldrect, oldmode, NULL, WP_DEFAULT, OP_NONE);
```

I'm not really sure why we are locking errors here.